src/vcbuild/dmd_backend.vcxproj

```diff
@@ -113,7 +113,7 @@
       <ExcludedFromBuild>true</ExcludedFromBuild>
     </ClCompile>
     <ClCompile Include="..\dmd\root\newdelete.c" />
-    <CustomBuild Include="..\dmd\backend\optabgen.c">
+    <CustomBuild Include="..\dmd\backend\optabgen.d">
       <Message>Building and running $(IntDir)%(Filename).exe</Message>
       <Command>cl /TP /I..\dmd\tk /I..\dmd\root /I. /I.. /FIwarnings.h /Fo"$(IntDir)%(Filename).obj" /Fe"$(IntDir)generated\%(Filename).exe" "%(FullPath)"
```

Comment:
Presumably you would have to change this command to a `$HOST_DC` equivalent one.